IdentityCore/src/cache/crypto/MSIDWPJKeyPairWithCert.h

```diff
@@ -41,6 +41,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) NSData *certificateData;
 @property (nonatomic, readonly) NSString *certificateSubject;
 @property (nonatomic, readonly) NSString *certificateIssuer;
+@property (nonatomic) NSString *keyChainVersion;
```

Comment:
If a user has registered only with Authenticator and not Company portal, then we would want keyChainVersion to be "v2" but if user then registers to Company portal, when the getWPJMetadata API is called we would want to over-write keyChain version from "v2" to "v1" since the entry would be available in "v1" now. So, kept it mutable.

Do you see any concerns around this and would prefer it to be readonly?